tests/aggregation/tests.py

```diff
@@ -1151,6 +1151,14 @@ def test_aggregation_subquery_annotation_exists(self):
         )
         self.assertTrue(publisher_qs.exists())
 
+    def test_aggregation_exists_annotation(self):
+        published_books = Book.objects.filter(publisher=OuterRef('pk'))
+        publisher_qs = Publisher.objects.annotate(
+            published_book=Exists(published_books),
+            count=Count('book'),
+        ).values_list('name', 'published_book')
+        self.assertTrue(publisher_qs.exists())
```

Comment:
Per new code guidelines, can we use `assertIs`? :)

Reply:
TBH I added `.exists()` only to evaluate queryset, this issue is not related with `exists()`, so I will change this assertion.

Reply:
Updated.